integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -63,24 +65,116 @@ If this does not suit your needs and you want to experiment with different promp
+
+### Configure the Milvus semantic search extension
+
+Milvus is an open-source vector database for semantic search.
+
+1. Follow the instructions at [Milvus](https://milvus.io) to [install it](https://milvus.io/docs/install_standalone-docker.md) somewhere that your assistant can access it. You will need the server variables to connect it with your assistant. One option for where to host is an [IBM virtual server](https://www.ibm.com/cloud/virtual-servers). Another is [IBM code engine](https://www.ibm.com/cloud/code-engine). However, it can also be installed on a separate machine as long as it is connected to the internet. Alternatively, Milvus can be run on a different cloud such as AWS. You need to record the IP address of the machine where it's installed Milvus and the port where it is running.
+
+1. Add your content to Milvus. In this example, we use Milvus to search a document collection that we previously indexed with [this python script](index-with-milvus.py). We include this script as an example for reference. In our example we use `.pdf` documents; the script can be modified to use other text formats. We have not included sample documents for search because we encourage you to use your own and modify the script accordingly.
+
+1. Download [the Milvus OpenAPI specification file](milvus-openapi.json) from this starter kit. You use this specification file to create and add the Milvus extension to your assistant.
+
+1. In your assistant, on the **Integrations** page, click **Build custom extension** and use the OpenAPI specification file to build a custom extension named `Milvus`. For general instructions on building any custom extension, see [Building the custom extension](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension).
+
+1. After you build the Milvus extension and it appears on your **Integrations** page, click **Add** to add it to your assistant. For general instructions on adding any custom extension, see [Adding an extension to your assistant](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-add-custom-extension).
+
+1. In **Authentication**, choose **No authentication** and enter values for your Milvus server variables to create a valid generated URL for requests. However, if you configured Milvus to require basic authentication, then choose **Basic auth** and enter your Milvus credentials.
+
+1. The setup of your Milvus extension for Watson Assistant is complete. For more information about the extension parameters, see the [Milvus API Reference](https://milvus.io/docs/search.md).
+
+### Configure the OpenAI answer generation extension
+
+Follow the steps in the starter kit for [language model openAI](../language-model-openai/README.md) to get an OpenAI api key and use it to connect with your assistant via a custom extension.
+
+### Upload sample action
+
+The starter kit includes a JSON file with these sample actions:
```

Comment:
Can you specify the name of that file & link here too? And possibly you can remove that from the later step.

Reply:
Done!